### PR TITLE
Maven build fix

### PR DESCRIPTION
**Pull request by kevinhinterlong**
Use maven to run setup tasks instead of a bash file.
[Source](https://github.com/yahoo/fili/pull/144/files#diff-f3da144ad2acdd85e52867800b51d0d3)

### CODE REVIEW

File: fili-system-config/pom.xml [hunk 1]
```diff
@@ -80,4 +106,4 @@
             </plugin>
         </plugins>
     </build>
-</project>
+</project>
```
Comment (cdeszaq) [inline fili-system-config/pom.xml]: Somewhere you lost the trailing newine

Comment (kevinhinterlong) [inline fili-system-config/pom.xml]: 👍

File: fili-system-config/pom.xml [hunk 2]
```diff
@@ -48,11 +48,17 @@
...
                         <configuration>
                             <tasks>
-                                <exec
-                                    dir="${project.basedir}/src/test/scripts"
-                                    executable="./prepjars.bash"
-                                    failonerror="true">
-                                </exec>
+                                <delete
+                                        quiet="true"
+                                        file="${project.build.directory}/moduleJars"/>
```
Comment (garyluoex) [inline fili-system-config/pom.xml]: Should this parameter be `file` or `dir`?

File: fili-system-config/pom.xml [hunk 3]
```diff
@@ -62,15 +68,35 @@
...
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <additionalClasspathElements>
                         <!-- One set of paths makes maven happy, the other set makes the IDE happy -->
-                        <additionalClasspathElement>${basedir}/src/test/resources/jars/fili-system-config-test1.jar</additionalClasspathElement>
-                        <additionalClasspathElement>${basedir}/src/test/resources/jars/fili-system-config-test2.jar</additionalClasspathElement>
-                        <additionalClasspathElement>src/test/resources/jars/fili-system-config-test1.jar</additionalClasspathElement>
-                        <additionalClasspathElement>src/test/resources/jars/fili-system-config-test2.jar</additionalClasspathElement>
+                        <additionalClasspathElement>${basedir}/target/moduleJars/jars/fili-system-config-test1.jar</additionalClasspathElement>
+                        <additionalClasspathElement>${basedir}/target/moduleJars/jars/fili-system-config-test2.jar</additionalClasspathElement>
```
Comment (garyluoex) [inline fili-system-config/pom.xml]: We should be consistent to use `${project.build.directory}` here, it seems like `${basedir}` is deprecated now.

File: fili-system-config/pom.xml (hunk 2 above)
Comment (kevinhinterlong) [inline fili-system-config/pom.xml]: 👍 dir

File: fili-system-config/pom.xml (hunk 3 above)
Comment (kevinhinterlong) [inline fili-system-config/pom.xml]: 👍 `${project.basedir}`

**Pull request merged**